### PR TITLE
Make it a package

### PR DESCRIPTION
**Pull request by szepeviktor**
This is example code.

The shim for non-Composer installations is not implemented.

### CODE REVIEW

**Comment by szepeviktor** [comment]
Apparently it gets installed in a subdirectory under `wp-content/mu-plugins/`.
So the provided shim can be a loader too: `wp-content/mu-plugins/wp-rest-api-auth0-loader.php`

- Detect global Composer
- Or require `wp-content/mu-plugins/wp-rest-api-auth0-loader/vendor/autoload.php`
- And load the MU plugin: `wp-content/mu-plugins/wp-rest-api-auth0-loader/wp-rest-api-auth0-loader.php`
-

**Comment by szepeviktor** [comment]
Loader added 🎉

**Comment by joshcanhelp** [comment]
@szepeviktor - I appreciate this greatly, thank you! I'm just about to publish the post for this and I'll take all this into account 👍

**Comment by szepeviktor** [comment]
You're welcome.

My clients do not mean technical challenges to me so I run [theoretical researches](https://github.com/szepeviktor/small-project/blob/master/MAIN-FILE-PARTS.md) and send this kind of PR-s.

File: README.md [hunk 1]
```diff
@@ -13,4 +30,8 @@ This repo is more of an example rather than a deployable piece of software. To g
 1. Move those `define()` statements from `env.example.php` to your `wp-config.php` and define them
 1. Move `wp-rest-api-auth0.php` and the `wp-rest-api-auth0` directory into the `wp-content/mu-plugins` directory in the WordPress install (make one if it does exist)
 
+## Testing with Docker
+
 You can get this running to test using Docker [using this Gist](https://gist.github.com/joshcanhelp/0e35b657ca03142e3d79595c28bb3ed7).
+
+TODO Move that gist into this repo.
```
Comment (joshcanhelp) [inline README.md:37]:
```suggestion
```

**Review by joshcanhelp — APPROVED** [review]
LGTM! Thanks again!

**Pull request merged**